integration/auth/auth_test.go

```diff
@@ -161,6 +175,23 @@ func TestRevokeRefreshTokens(t *testing.T) {
 	}
 }
 
+func verifyCustomToken(t *testing.T, ct, uid string) *auth.Token {
+	idt, err := signInWithCustomToken(ct)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer deleteUser(uid)
```

Comment:
I would add a comment for this function clarifying that, in addition to verifying the token, this function creates and deletes the user with the given UID.

You know, it might actually make more sense to get the uid out of `ct`, if possible. Otherwise, there's the small chance of passing in the wrong uid.

Reply:
Added comment. 

I'd prefer to avoid the token decoding step necessary to extract the uid from the custom token here. If the wrong uid gets passed in, the test will fail since we compare them later.